site/src/patterns/card.md

```diff
@@ -48,28 +48,86 @@ This card type contains a super title labelled by the card's type, a title, an i
+				</div>
+				<a class="text-decoration-none justify-self-stretch stretched-link margin-top-auto" href="#"
+					>Watch videos</a
+				>
+			</div>
+		</div>
+	</div>
+</article>
+```
+
+## Summary card
+
+These cards use standard `.card` subcomponents, border atomics to create a left border, and (because they have only one clickable item inside) the [stretched-link component](../components/stretched-link.md). They use `.card-content-description` to truncate a string of summary text. Note that these will typically not be full-width elements, but will be displayed side-by-side with other cards of this type.
+
+```html
+<article class="card position-relative border-left-lg border-left-color-accent">
 	<div class="card-content">
+		<a href="#" class="card-title stretched-link">The title of the card</a>
+		<p class="card-content-description">
+			The longer description of the card, which will eventually become truncated the more we repeat
+			this sentence. The longer description of the card, which will eventually become truncated the
+			more we repeat this sentence. The longer description of the card, which will eventually become
+			truncated the more we repeat this sentence. The longer description of the card, which will
+			eventually become truncated the more we repeat this sentence.
+		</p>
+	</div>
+</article>
+```
+
+## Testimonial card
+
+The testimonial card provides a place for a user's opinion to be showcased. Notable modifications from Icon Card include using `border-radius-rounded` on the image, as well as slightly different font and text sizes. There are also several flexbox based atomics applied to ensure links justify at the ends of cards when placed in a grid.
+
+```html
+<article class="card">
+	<div class="card-content font-size-lg display-flex flex-direction-column">
 		<div class="media">
 			<div class="media-left">
 				<div class="image image-64x64">
-					<img src="~/src/scaffold/media/video-media.png" />
+					<img
+						class="border-radius-rounded"
+						alt="Write alt text here or if image is presentational only add aria-hidden attribute"
+						src="~/src/scaffold/media/twitter-example.jpeg"
+					/>
 				</div>
 			</div>
 			<div class="media-content">
-				<p class="text-transform-uppercase margin-bottom-xxs">Videos</p>
-				<p class="font-size-lg font-weight-semibold margin-bottom-xxs">
-					Media content is a place to put your text content.
-				</p>
-				<p class="color-text-subtle margin-bottom-xs">Videos to help you get started.</p>
-				<a class="text-decoration-none stretched-link" href="#">Watch videos</a>
+				<p class="font-size-xl font-weight-semibold margin-bottom-xxs">Full Name</p>
+				<p href="#">Role</p>
 			</div>
 		</div>
+		<p class="margin-block-xs color-text-subtle">
+			"Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor."
+		</p>
+		<a href="#" class="display-block text-decoration-none margin-top-auto font-size-md"
```

Comment:
It keeps bothering me to not see the underline when I'm hovering over the link. I also see that in docs all anchor tags don't have the underline on the default case, which makes `text-decoration-none` unnecessary here, but it is in the atlas. Should it be synced?